spacialgaze-plugins/misc.js

```diff
@@ -372,6 +372,13 @@ exports.commands = {
 			msg += '<button class="button" name="send" value="/emote add, ' + target[1] + ', ' + target[2] + '">Add emote</button></center>';
 			delete user.tokens[target[0]];
 			return SG.messageSeniorStaff(msg);
+		case 'disableintroscroll':
```

Comment:
Ok for this though, when the user tries to use the token check to make sure the room exists, and the room dosent already have its roomintro limit removed.